examples/plaintext.rs

```diff
@@ -0,0 +1,54 @@
+use std::net::TcpStream;
+
+fn main() {
+    // REMINDER this is unsafe, the credentials are sent over the connection in CLEARTEXT
```

Comment:
The word "unsafe" has a very particular meaning in the Rust ecosystem. I suggest we go with "dangerous" instead.